coalib/processes/Processing.py

```diff
@@ -405,14 +417,6 @@ def instantiate_processes(section,
                         'control_queue': control_queue,
                         'timeout': 0.1}
 
-    local_bear_list[:], global_bear_list[:] = instantiate_bears(
-        section,
-        local_bear_list,
-        global_bear_list,
-        complete_file_dict,
-        message_queue,
-        console_printer=console_printer)
-
```

Comment:
I moved everything i could move. The file_dict variable uses the result of caching with the filename_list variable